Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.13 as builder
+WORKDIR /go/src/github.com/openshift/bond-cni
+COPY . .
+RUN ./build.sh
+
+FROM alpine:latest
```

Comment:
please use a centos image here not alpine

Reply:
Done